services/movie.py

```diff
@@ -0,0 +1,30 @@
+from db.models import Movie
+from django.db.models import QuerySet
+
+
+def get_movies(genres_ids: list = None, actors_ids: list = None) -> QuerySet:

```

Comment:
You should specify inenr type of `list` and `QuerySet`. Also, if argument can be `None` you should show it in type hint. Fix everywhere, please
```suggestion
def get_movies(genres_ids: list[int] | None = None, actors_ids: list[int] | None = None) -> QuerySet[Movie]:

```